lighthouse-core/test/report/html/renderer/util-test.js

```diff
@@ -53,4 +53,16 @@ describe('util helpers', () => {
     assert.equal(Util.calculateRating(0.80), 'pass');
     assert.equal(Util.calculateRating(1.00), 'pass');
   });
+
+  it('formats display values', () => {
+    const format = (...args) => Util.formatDisplayValue(...args);
```

Comment:
don't need to rest and then spread since they're all arrays below?

Reply:
done